operations/template/app.tf

```diff
@@ -227,9 +221,18 @@ resource "azurerm_linux_web_app_slot" "pre_live" {
   }
 
   app_settings = {
-    DOCKER_REGISTRY_SERVER_URL = "https://${azurerm_container_registry.registry.login_server}"
-
-    ENV = var.environment
+    DOCKER_REGISTRY_SERVER_URL    = "https://${azurerm_container_registry.registry.login_server}"
+    ENV                           = var.environment
+    REPORT_STREAM_URL_PREFIX      = "https://${local.rs_domain_prefix}prime.cdc.gov"
+    KEY_VAULT_NAME                = azurerm_key_vault.key_storage.name
```

Comment:
Are we not using sticky settings now at all?

Reply:
When we didn't share all the settings, we were getting errors in `pre-live` after swapping (which would prevent swapping back). Since TI is driven off of http calls, it seemed simpler to share all the settings than to figure out which ones are required for the app to be healthy, which makes the sticky settings unneeded for TI